src/components/CommentCard.jsx

```diff
@@ -0,0 +1,46 @@
+
+import { useState, useEffect } from "react";
+import { getAllComments } from "../api";
+import { Axios } from "axios";
+import SingleReview from '../components/SingleReview'
+import { formatDate } from "../utils";
+
+
+const CommentCard = ({review, commentSection, setCommentSection, commentShow}) => {
+    
+
+    useEffect(() => {
+        getAllComments(review).then(({ comments } ) => {
+       
+           setCommentSection(comments)
+       }) 
+    }, [commentShow])
```

Comment:
This is nice - 

- [ ] could we have some loading state, just incase they open they show the comments straight away, or the api takes it time to fetch the data?